libs/blocks/video/video.css

```diff
@@ -5,4 +5,142 @@ a[href*='.mp4'].hide-video {
 video {
   max-width: 100%;
   height: auto;
+  object-fit: cover;
+}
+
+:is(.marquee, .aside.split) .pause-play-wrapper img.accessibility-control {
+  min-height: 40px;
+}
+
+.video-container {
+  display: flex;
+  position: relative;
+  height: 100%;
+  width: fit-content;
+  margin: auto;
+}
+
+:is(.aside, .marquee, .quiz-marquee) .video-container {
+  width: auto;
+}
+
+.brick-media .video-container {
+  width: 100%;
+  height: 100%;
+  border-radius: inherit;
+}
+
+.pause-play-wrapper {
+  display: flex;
+  width: fit-content;
+  margin: auto;
+}
+
+.video-container .pause-play-wrapper {
+  width: 45px !important;
+  height: 45px !important;
+  position: absolute;
+  bottom: 2%;
+  right: 2%;
```

Comment:
Did we get any directions for RTL languages? Right now the position is unchanged (bottom right side)

Reply:
yes, The buttons are placed on the left and right based on which side the video is placed no matter the direction of the page.